pgqueuer/cli.py

```diff
@@ -54,6 +55,7 @@ async def display_pg_channel(
         connection,
         channel,
         defaultdict(lambda: deque(maxlen=1)),
+        defaultdict(anyio.CancelScope),
```

Comment:
This seems inconsistent with the usage in QM where this is a `dict[JobId, Context]`